manifests/40-rbac.yaml

```diff
@@ -34,10 +34,10 @@ rules:
 - apiGroups: ["security.openshift.io"]
```

Comment:
by adopting SDK/Kubebuilder you will be able to work with makers, e.g: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/controllers/memcached_controller.go#L44-L48

Then, when you run `make generate` the RBAC will be generated at the[config](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config)/rbac/ dir: https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config/rbac

Also, you can use `make bundle` and have the whole OLM bundle generated by you with all your kustomize configs, see: https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/bundle

That is very helpful to work with the releases and provide the solutions for OLM.
You can also add customizations on in your base CSV: [operator-sdk](https://github.com/operator-framework/operator-sdk)/[testdata](https://github.com/operator-framework/operator-sdk/tree/master/testdata)/[go](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go)/[v3](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3)/[memcached-operator](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator)/[config](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config)/[manifests](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config/manifests)/bases/